api/jobset/v1alpha2/jobset_types.go

```diff
@@ -74,6 +74,9 @@ type JobSetStatus struct {
 	// Restarts tracks the number of times the JobSet has restarted (i.e. recreated in case of RecreateAll policy).
 	Restarts int `json:"restarts,omitempty"`
 
+	// Active shows if there is at least one active child task
+	Active bool `json:"active,omitempty"`
```

Comment:
This is not useful on of its own for JobSet users. But you can add the Active count under `ReplicatedJobStatus` similar to ready/succeeded/failed.

Reply:
as I know, @mcariatm is going to use this property in kueue integration here: 
https://github.com/kubernetes-sigs/kueue/pull/762/files#diff-d3853fca35347a8a9499f4941707f191a6674df29274b49a9b4ec568eacb0605R79